.taskcluster.yml

```diff
@@ -40,9 +40,8 @@ tasks:
               git fetch ${repository} ${event.pull_request.head.ref}
               && git config advice.detachedHead false
               && git checkout ${event.pull_request.head.sha}
-              && rm -rf gvr-android-sdk && git clone https://github.com/MozillaReality/FirefoxReality-gvr-android-sdk.git gvr-android-sdk
               && git submodule update
-              && ./gradlew --no-daemon --console=plain clean `python tools/taskcluster/build_targets.py =all+googlevr+noapi`
+              && ./gradlew --no-daemon --console=plain clean `python tools/taskcluster/build_targets.py =all+noapi=all`
```

Comment:
`=all+noapi=all` Is that final `=all` right or is a typo?

Reply:
It is correct. Otherwise the x86_64 doesn't build.